control-plane/subcommand/partition-init/command_ent_test.go

```diff
@@ -0,0 +1,74 @@
+//go:build enterprise
```

Comment:
do we need this one? The others don't have that.

Reply:
yeah. that happened accidently.